src/rpc/client/ts/package.json

```diff
@@ -16,6 +16,7 @@
     "compile-src": "run-script-os",
     "compile-src:default": "./compile-proto.sh",
     "compile-src:windows": "./compile-proto.bat",
+    "precompile": "(echo \"// GENERATED SOURCE -- DO NOT EDIT\"; node -p \"'export const OMEGA_EDIT_CLIENT_VERSION = ' + JSON.stringify(require('./package.json').version)\") > src/client_version.ts",
     "compile": "tsc -p ./",
```

Comment:
You will need to add a `pretest` script to the package.json to compile the code before testing. Something like this

```json
"pretest": "yarn compile"
```

Reply:
Good catch!